src/clj/vanilla/db/core.clj

```diff
@@ -202,7 +231,7 @@
   (.exists (jio/file (database :dbname))))
 
 
-
+;@TODO - is this still needed?
 (defn setup-database
```

Comment:
We should probably go through and just cleanup all these comments and add in any new ones that would be helpful for the postgres database.  I dont think its worth keeping this much deprecated code around/commented.  We're postgres now!  And if we need it, its always in git